test_conformance/extensions/cl_khr_command_buffer/negative_command_buffer_svm_mem.cpp

```diff
@@ -0,0 +1,413 @@
+        return BasicSVMCommandBufferTest::Skip()
+            || is_extension_available(device,
+                                      "cl_khr_command_buffer_multi_device");
+    }
+
+    const cl_char pattern_1 = 0x14;
+};
+
+// CL_INVALID_OPERATION if command_buffer has been finalized.
+struct CommandBufferCommandSVMFinalizedCommandBuffer
+    : public BasicSVMCommandBufferTest
+{
+    using BasicSVMCommandBufferTest::BasicSVMCommandBufferTest;
+
+    cl_int Run() override
+    {
+        cl_int error = clFinalizeCommandBufferKHR(command_buffer);
+        test_error(error, "clFinalizeCommandBufferKHR failed");
+
+        error = clCommandSVMMemcpyKHR(command_buffer, nullptr, svm_out_mem(),
+                                      svm_in_mem(), data_size(), 0, nullptr,
+                                      nullptr, nullptr);
+
+        test_failure_error_ret(
+            error, CL_INVALID_OPERATION,
+            "clCommandSVMMemcpyKHR should return CL_INVALID_OPERATION",
+            TEST_FAIL);
+
+        error = clCommandSVMMemFillKHR(command_buffer, nullptr, svm_in_mem(),
+                                       &pattern_1, sizeof(cl_char), data_size(),
+                                       0, nullptr, nullptr, nullptr);
+
+        test_failure_error_ret(
+            error, CL_INVALID_OPERATION,
+            "clCommandSVMMemFillKHR should return CL_INVALID_OPERATION",
+            TEST_FAIL);
+
+        return CL_SUCCESS;
+    }
+
+    bool Skip() override
+    {
+        return BasicSVMCommandBufferTest::Skip()
+            || is_extension_available(device,
+                                      "cl_khr_command_buffer_multi_device");
+    }
+
+    const cl_char pattern_1 = 0x14;
+};
+
+// CL_INVALID_VALUE if mutable_handle is not NULL.
+struct CommandBufferCommandSVMMutableHandleNotNull
+    : public BasicSVMCommandBufferTest
+{
+    using BasicSVMCommandBufferTest::BasicSVMCommandBufferTest;
+
+    cl_int Run() override
+    {
+        cl_mutable_command_khr mutable_handle;
+
+        cl_int error = clCommandSVMMemcpyKHR(
+            command_buffer, nullptr, svm_out_mem(), svm_in_mem(), data_size(),
+            0, nullptr, nullptr, &mutable_handle);
+
+        test_failure_error_ret(
+            error, CL_INVALID_VALUE,
+            "clCommandSVMMemcpyKHR should return CL_INVALID_VALUE", TEST_FAIL);
+
+        error = clCommandSVMMemFillKHR(command_buffer, nullptr, svm_in_mem(),
+                                       &pattern_1, sizeof(cl_char), data_size(),
+                                       0, nullptr, nullptr, &mutable_handle);
+
+        test_failure_error_ret(
+            error, CL_INVALID_VALUE,
+            "clCommandSVMMemFillKHR should return CL_INVALID_VALUE", TEST_FAIL);
+
+        return CL_SUCCESS;
+    }
+
+    bool Skip() override
+    {
+        return BasicSVMCommandBufferTest::Skip()
+            || is_extension_available(device,
+                                      "cl_khr_command_buffer_multi_device");
```

Comment:
Even with the multidevice extension I think this would still be an error, so don't need to skip with the presence of this extension.

Reply:
@EwanC I will remove Skip() for this test.